ansible/regtest/playbooks/pcc/mass_install.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- hosts: server_emulator  #host details from which we are calling API, we are ruunnig on setup of invader28 i.e 172.17.2.28
+  become: true
+  tasks:
+    - name: Check that we can login to PCC   
+      uri:
+        url: https://172.17.2.215:9999/security/auth  #API for the PCC login.If you want to run this playbook some other PCC than you need to update the ip address of that PCC server in API
+        validate_certs: False
+        method: POST
+        body: {username: "{{ pcc_user }}", password: "{{ pcc_password }}"}
+        status_code: 200
+        body_format: json
+        return_content: yes
+
+
+    - name: Install a MaaS on node
+      uri:
+        url: https://172.17.2.215:9999/pccserver/node/update  #API for the PCC login.If you want to run this playbook some other PCC than you need to update the ip address of that PCC server in API
+        method: PUT
+        headers:
+          Authorization: Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZG1pbiIsImV4cCI6MTU2MDI0ODE3NCwiaWF0IjoxNTU5NjQzMzc0fQ.gfrLcI8cS-pRiHZfFfO-Ph-RHtD3veItr6k4CDNoRL51-miImnHIV6ThrbvB4g_yfIY7EIt3jdvTr_wRYkoGcA
```

Comment:
@sbpopade, why are hard coding the bearer token in the playbook. If it's need to be hard coded for some reason then use a variable instead.

Reply:
These are the temporary scripts we created , I will do the changes tomorrow if needed .

Reply:
We have removed all files regarding PCC.